### PR TITLE
Adds complementing text color for various brand colors that the user might choose

### PR DESCRIPTION
**Pull request by alishaz-polymath**
## What does this PR do?

- adds a complementing text color for brand colors based on their respective contrast to make sure that the text remains visible for any brand color that the end users might choose

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

- [x] Choose #ffffff as the brand color when background is dark
- [x] Choose #292929 as the brand color when background is light
- [x] Choose any random hex as the brand color and see if the text is visible

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code and corrected any misspellings
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by alishaz-polymath** [comment]
The Build/Check types errors:
  - components/ui/form/PhoneInput.tsx
  - pages/availability/troubleshoot.tsx
are unrelated to this PR

File: components/AddToHomescreen.tsx [hunk 1]
```diff
@@ -13,14 +13,14 @@ export default function AddToHomescreen() {
...
-      <div className="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8">
+    <div className="fixed inset-x-0 bottom-0 pb-2 sm:hidden sm:pb-5">
+      <div className="px-2 mx-auto max-w-7xl sm:px-6 lg:px-8">
         <div className="p-2 rounded-lg shadow-lg sm:p-3" style={{ background: "#2F333D" }}>
-          <div className="flex items-center justify-between flex-wrap">
-            <div className="w-0 flex-1 flex items-center">
-              <span className="flex p-2 rounded-lg bg-opacity-30 bg-brand">
+          <div className="flex flex-wrap items-center justify-between">
+            <div className="flex items-center flex-1 w-0">
+              <span className="flex p-2 rounded-lg bg-opacity-30 bg-brand text-brandtext">
```
Comment (PeerRich) [inline components/AddToHomescreen.tsx:21]: we could rename this to text-brand right?

Comment (zomars) [reply components/AddToHomescreen.tsx:21]: Agreed `text-brandtext` sounds really weird IMO.

**Review by zomars — APPROVED** [review]
Nice work, although I'm no 100% convinced about the naming for `text-brandtext`. Can we use something like `text-branding`?

Also we should add visual regression testing to the Backlog.

**Comment by alishaz-polymath** [comment]
> Nice work, although I'm no 100% convinced about the naming for `text-brandtext`. Can we use something like `text-branding`?
> 
> Also we should add visual regression testing to the Backlog.

Yeah, I think we can maybe think over it a bit. "branding" would be confusing though, since we already have "brand", so we perhaps need something that might make sense by itself. Perhaps something like "contrast" or "complement" so that the class becomes "text-contrast" or "text-complement"?

**Comment by PeerRich** [comment]
> > Nice work, although I'm no 100% convinced about the naming for `text-brandtext`. Can we use something like `text-branding`?
> 
> > 
> 
> > Also we should add visual regression testing to the Backlog.
> 
> 
> 
> Yeah, I think we can maybe think over it a bit. "branding" would be confusing though, since we already have "brand", so we perhaps need something that might make sense by itself. Perhaps something like "brand-contrast" or "complement" so that the class becomes "text-brand-contrast" or "text-complement"? 
> 
> 

i like text-brand-contrast

**Comment by alishaz-polymath** [comment]
> > > Nice work, although I'm no 100% convinced about the naming for `text-brandtext`. Can we use something like `text-branding`?
> > 
> > 
> > > 
> > 
> > 
> > > Also we should add visual regression testing to the Backlog.
> > 
> > 
> > Yeah, I think we can maybe think over it a bit. "branding" would be confusing though, since we already have "brand", so we perhaps need something that might make sense by itself. Perhaps something like "brand-contrast" or "complement" so that the class becomes "text-brand-contrast" or "text-complement"?
> 
> i like text-brand-contrast

Actually, we can't have brand-contrast as the class. It would have to be a single word, so "brandcontrast" instead. But I think even this makes more sense than what we have right now.

**Comment by PeerRich** [comment]
> Actually, we can't have brand-contrast as the class. It would have to be a single word, so "brandcontrast" instead. But I think even this makes more sense than what we have right now.

ok go for it

**Review by PeerRich — APPROVED** [review]

**Comment by alishaz-polymath** [comment]
[Note: please don't merge yet. More changes coming up]

**Comment by PeerRich** [comment]
i bet the time picker being fixed, too?
image

**Pull request merged**